package-lock.json

```diff
@@ -17745,9 +17794,17 @@
 			"version": "file:packages/redux-routine",
 			"requires": {
 				"@babel/runtime": "^7.16.0",
+				"is-plain-obj": "^4.1.0",
 				"is-promise": "^4.0.0",
 				"lodash": "^4.17.21",
 				"rungen": "^0.3.2"
+			},
+			"dependencies": {
```

Comment:
It would be great to deduplicate `is-plain-obj`. At the moment, it will install several copies in every WP package that uses it.

Reply:
Indeed, although IMHO this is separate from this PR, it should be addressed as part of the work remaining in https://github.com/WordPress/gutenberg/issues/17025#issuecomment-1186883437

Or do you have anything else in mind?

Reply:
If you revert all changes from this branch to `package-lock.json` and install `is-plain-obj` for the whole project then it will add only one copy. Then you only need to remove it from the main `package.json` and it will remain correctly placed.

Reply:
Just pushed what you suggested. However, it seems like `minimist-options` depends on `is-plain-obj@^1.1.0` so it will still end up in multiple copies because we end up using different versions. Or did you have something else in mind?

Reply:
That's perfectly fine when some packages require a different older version 👍🏻

Reply:
Cool, thanks for your help @gziolo 🙌